eahub/base/static/components/feedback.js

```diff
@@ -0,0 +1,26 @@
+async function ajaxsubmit(){
+  document.getElementById('page_url').value = window.location.href;
```

Comment:
I think it's better to either do that on the backend or replace with `window.location.pathname`, because `href` adds any user specific parameters that will look odd in admin, eg

window.location.href
"http://127.0.0.1:8000/profiles/?toggle%5Bavailable_to_volunteer%5D=true&refinementList%5Bcountry%5D%5B0%5D=USA"

Reply:
while window.location.pathname is `/profiles/`/profiles/